applications/tari_app_grpc/src/blocks.rs

```diff
@@ -63,10 +63,11 @@ impl From<tari_core::blocks::Block> for grpc::Block {
                     .iter()
                     .map(|input| grpc::TransactionInput {
                         features: Some(grpc::OutputFeatures {
-                            flags: input.features.flags.bits() as u32,
-                            maturity: input.features.maturity,
+                            flags: input.features().flags.bits() as u32,
+                            maturity: input.features().maturity,
                         }),
-                        commitment: Vec::from(input.commitment.as_bytes()),
+                        commitment: Vec::from(input.commitment().as_bytes()),
```

Comment:
nit: (In a few places)
```suggestion
                        commitment: input.commitment().to_vec(),
```